### PR TITLE
hotfix invitation link

### LINKED ISSUE #1603: invitation doesn't work

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #1603

### CODE REVIEW

File: src/server/service/mail/userInvite.ts [hunk 1]
```diff
@@ -16,9 +16,10 @@ export const userInvite = async (props: {
 
   const i18n = await createI18nPromise('en')
 
-  const link = `${url}/login${role.invitationUuid ? `?invitationUuid=${role.invitationUuid}` : ''}`
+  const link = `${url}/login/invitation${role.invitationUuid ? `?invitationUuid=${role.invitationUuid}` : ''}`
```
Comment (minotogna) [inline src/server/service/mail/userInvite.ts]: can we use BasePaths ?

File: src/client/components/UserList/UserInvitationInfo/UserInvitationInfo.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const UserInvitationInfo: React.FC<{ user: User; onClose: () => void }> = ({ use
         <div>
           {`${i18n.t('userManagement.invitationLink')}: ${
             window.location.origin
-          }/login?invitationUuid=${invitationUuid}`}
+          }/login/invitation?invitationUuid=${invitationUuid}`}
```
Comment (minotogna) [inline src/client/components/UserList/UserInvitationInfo/UserInvitationInfo.tsx]: can we use BasePaths ?

File: src/meta/user/users.ts [hunk 3]
```diff
@@ -35,17 +36,17 @@ const getRolesAllowedToEdit = (props: { user: User; countryIso: CountryIso }): A
...
-      RoleName.VIEWER,
     ]
   }
 
   if (isNationalCorrespondent(user, countryIso) || isAlternateNationalCorrespondent(user, countryIso)) {
-    return [RoleName.COLLABORATOR, RoleName.VIEWER]
+    return [RoleName.COLLABORATOR]
   }
   return []
 }
```
Comment (minotogna) [inline src/meta/user/users.ts]: why removing RoleName.VIEWER?

File: src/meta/basePaths.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export const BasePaths = {
+  Root: () => '/',
```
Comment (sorja) [inline src/meta/basePaths.ts]: Should we have all paths here?

Comment (minotogna) [reply src/meta/basePaths.ts]: We should have only one BasePaths

File: src/server/service/mail/userInvite.ts [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import { createI18nPromise } from '@i18n/i18nFactory'
 
 import { CountryIso } from '@meta/area'
+import { BasePaths } from '@meta/basePaths'
```
Comment (minotogna) [inline src/server/service/mail/userInvite.ts]: What about @meta/app ?
So we'll have only one BasePaths 
Import {BasePaths} from meta/app .
What do you @sorja @rfontanarosa

**Review by minotogna — CHANGES_REQUESTED** [review]

File: src/server/service/mail/userInvite.ts (hunk 5 above)
Comment (rfontanarosa) [reply src/server/service/mail/userInvite.ts]: ok for me

Comment (sorja) [reply src/server/service/mail/userInvite.ts]: ok

**Review by sorja — APPROVED** [review]

**Review by minotogna — APPROVED** [review]
👍

**Pull request merged**